edgelet/iotedged/src/lib.rs

```diff
@@ -402,6 +402,14 @@ where
 
                         tokio_runtime.block_on(reprovision)?;
 
+                        // After reprovisioning, previous module keys may no longer be valid.
```

Comment:
Merge https://github.com/Azure/iotedge/blob/release/1.1/edgelet/iotedged/src/lib.rs#L359-L363 into this instead of deleting twice.

Reply:
Is that possible? It seems like L359-L356 needs to happen before the call to start_api, while this code (L405) can only happen after start_api returns because it depends on the return value of start_api.

Reply:
Oh right, `start_api` blocks until it's time to exit.